x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -338,6 +342,43 @@ export const ElasticsearchOverview = () => {
           })}
         />
       </EuiPageTemplate.Section>
+      <EuiPageTemplate.Section
+        color="subdued"
+        bottomBorder="extended"
+        data-test-subj="pipeline-client-section"
+      >
+        <OverviewPanel
+          description={
+            <FormattedMessage
+              id="xpack.serverlessSearch.pipeline.description"
+              defaultMessage="Use {ingestPipelinesLink} to preprocess your data before it's indexed into Elasticsearch, which is often much easier than post-processing. Use any combination of ingest processors to add, delete, or transform fields in your documents."
+              values={{
+                ingestPipelinesLink: (
+                  <EuiLink
+                    data-test-subj="serverlessSearchElasticsearchOverviewIngestPipelinesLink"
+                    href={
+                      'https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html'
+                    }
```

Comment:
Few questions:
1. Do I have to add `target="_blank"` for this link?
2. Does it needs to be coming from `docLinks`? If yes, does it require a separate PR for package change or can be integrated in the same PR? The link exists in `kibana/packages/kbn-doc-links/src/get_doc_links.ts`.